docker/local/Dockerfile.web

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-bullseye
+FROM python:3.10
```

Comment:
Warum nicht das 3.10-slim-bullseye ? 
Kleineres Image

Reply:
mit der `3.10-slim-bullseye` läuft `weasyprint` nicht, aber ich kann mal probieren welche kleineren Images alternativ laufen würden

Reply:
also mit `slim` bzw `slim-bullseye` bzw `slim-bookworm` läuft das nicht:
`OSError: cannot load library 'gobject-2.0-0': gobject-2.0-0: cannot open shared object file: No such file or directory.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'gobject-2.0-0'`

Mit `3.10.-bullseye` oder auch mit `3.10-bookworm` keine Probleme, kann ansonsten noch mal testen was mit `alpine` geht

Reply:
Naja `bookworm` ist Debian 12 und `bullseye` ist Debian 11, das sind details die wir glaube ich gar nicht berücksichtigen

Reply:
Also bei mir funktioniert `3.10-slim` in `web`und `celery-worker`.
Seltsamer weise fehlt im `Dockercile.celery` aber der install vom `wkhtmltopdf`